### ISSUE TITLE
UselessParentheses false positive new static($var)

### ISSUE THREAD

**Issue by Levivb** [slevomat/coding-standard#404]
Hey,

I encountered a false positive on the `UselessParenthesesSniff` in the following snippet:
```php
new static($var);
```
It is fixed to
```php
new static$var
```
Same goes for
```php
new self($var)
```

Hope you can get around to fixing it 👍 

Keep up the good work!

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/919b593a48aa7b519c12ec639fb03aeb6b102bb2
